credentials/generate-revocation-set.py

```diff
@@ -133,99 +170,119 @@ def main(use_main_net_dcld, use_test_net_dcld, use_main_net_http, use_test_net_h
         crl_authority_key_id = crl_file.extensions.get_extension_for_oid(x509.OID_AUTHORITY_KEY_IDENTIFIER).value.key_identifier
         crl_signer_subject_key_id = crl_signer_certificate.extensions.get_extension_for_oid(
             x509.OID_SUBJECT_KEY_IDENTIFIER).value.key_identifier
         if crl_authority_key_id != crl_signer_subject_key_id:
+            print("CRL Authority Key ID is not CRL Signer Subject Key ID, continue...")
             continue
 
         issuer_subject_key_id = ''.join('{:02X}'.format(x) for x in crl_authority_key_id)
 
-        same_issuer_points = None
+        # b.
+        same_issuer_points = dcld_helper.get_revocations_points_by_skid(issuer_subject_key_id)
+        count_with_matching_vid_issuer_skid = sum(item.get('vid') == vid for item in same_issuer_points)
 
-        # TODO: Extract this to a helper function
-        if use_rest:
-            response = requests.get(
-                f"{rest_node_url}/dcl/pki/revocation-points/{issuer_subject_key_id}").json()["pkiRevocationDistributionPointsByIssuerSubjectKeyID"]
-            same_issuer_points = response["points"]
-        else:
-            cmdlist = ['query', 'pki', 'revocation-points', '--issuer-subject-key-id', issuer_subject_key_id]
-            cmdpipe = subprocess.Popen(use_dcld(dcld, production, cmdlist), stdout=subprocess.PIPE, stderr=subprocess.PIPE)
-            same_issuer_points = json.loads(cmdpipe.stdout.read())[
-                "pkiRevocationDistributionPointsByIssuerSubjectKeyID"]["points"]
-
-        matching_entries = False
-        for same_issuer_point in same_issuer_points:
-            if same_issuer_point["vid"] == vid:
-                matching_entries = True
-                break
-
-        if matching_entries:
+        if count_with_matching_vid_issuer_skid > 1:
             try:
                 issuing_distribution_point = crl_file.extensions.get_extension_for_oid(
                     x509.OID_ISSUING_DISTRIBUTION_POINT).value
             except Exception:
+                print("CRL Issuing Distribution Point not found, continue...")
                 continue
 
             uri_list = issuing_distribution_point.full_name
             if len(uri_list) == 1 and isinstance(uri_list[0], x509.UniformResourceIdentifier):
                 if uri_list[0].value != revocation_point["dataURL"]:
+                    print("CRL Issuing Distribution Point URI is not CRL URL, continue...")
                     continue
             else:
+                print("CRL Issuing Distribution Point URI is not CRL URL, continue...")
                 continue
 
         # 9. Assign CRL File Issuer
         certificate_authority_name = base64.b64encode(crl_file.issuer.public_bytes()).decode('utf-8')
+        print(f"CRL File Issuer: {certificate_authority_name}")
 
         serialnumber_list = []
         # 10. Iterate through the Revoked Certificates List
         for revoked_cert in crl_file:
+            print(revoked_cert)
```

Comment:
This is too verbose. Please remove, or add a `verbose_logging` bool member to self